hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/RingbufferContainer.java

```diff
@@ -331,6 +340,9 @@ public Data readAsData(long sequence) {
     }
 
     /**
+     * Reads multiple items from the ring buffer and returns the stored format without serialisation.
```

Comment:
Doesn't this method return a `long`? What do you mean by `without serialisation`?